app/assets/javascripts/libfigure/words.js

```diff
@@ -97,6 +97,7 @@ Tag.prototype.flatten = function (format) {
   if (format === FLATTEN_FORMAT_UNSAFE_TEXT || format === FLATTEN_FORMAT_SAFE_TEXT) {
     return flattenWordNode(this.body, format);
   } else if (format === FLATTEN_FORMAT_MARKDOWN || format === FLATTEN_FORMAT_HTML) {
+    Object.keys(this.attrs).length === 0 || throw_up('attrs not yet implemented, but used');
```

Comment:
unrelated to this PR, but I recently zapped a comment here that I miss having. This is better still.